lib/std/builtin.zig

```diff
@@ -55,7 +55,7 @@ pub const StackTrace = struct {
         };
         const tty_config = std.io.tty.detectConfig(std.io.getStdErr());
         try writer.writeAll("\n");
-        std.debug.writeStackTrace(self, writer, arena.allocator(), debug_info, tty_config) catch |err| {
+        std.debug.writeStackTrace(self, writer, debug_info, tty_config) catch |err| {
```

Comment:
the arena itself can be removed too

Reply:
i didn't notice that! fixed in 0bccadf28be8fe498f5de324095f09fec58169a3